packages/api/tests/lib/test-granule-remove-from-cmr.js

```diff
@@ -36,44 +31,6 @@ test.before(async (t) => {
   process.env.GranulesTable = randomString();
   await new Granule().createTable();
 
-  testCumulusMessage = {
```

Comment:
This is all just cleanup. Not related to the functional changes in this PR.

Reply:
What caused the cleanup?

Reply:
Yeah, this was a test I wrote in my previous Granule Delete PR. There were a couple points during that review where I moved tests around between files, split tests up among different files, and generally copy-pasted things around. This stuff was cruft left over, likely because it was copied from somewhere else where it was actually used. Here it's not necessary.